tests/test_gpu_openacc.py

```diff
@@ -227,12 +226,14 @@ def iso_acoustic(self, **opt_options):
 
         assert np.isclose(norm(rec), 490.56, atol=1e-2, rtol=0)
 
-    @pytest.mark.parametrize('opt_options', [
-        {},
-        {'linearize': True},
+    @pytest.mark.parametrize('opt', [
+        ('advanced', {}),
+        ('advanced', {'linearize': True}),
```

Comment:
this one is probably overkill (never forget that running tests has a cost) provided that we keep the last one, ie `(blocking, {linearize=true})`